src/stories/dataset/dataset-action-buttons/DatasetActionButtons.stories.tsx

```diff
@@ -27,15 +27,14 @@ export const WithPublishPermissions: Story = {
     <DatasetActionButtons
       dataset={DatasetMother.create({
         permissions: DatasetPermissionsMother.createWithAllAllowed(),
-        version: DatasetVersionMother.createDraftAsLatestVersion(),
+        version: DatasetVersionMother.createDraftAsLatestVersionWithSomeVersionHasBeenReleased(),
```

Comment:
Can you walk me through what this method does? I would like to try for a shorter name that is still as descriptive.

Reply:
This method creates a mock of the dataset version, the name tries to describe that this dataset version is a draft and it's the latest version available of the dataset + some version of the dataset has been released

This means that the dataset is published and the retrieved version is a draft which is also the latest version available